pkg/oc/bootstrap/docker/up.go

```diff
@@ -242,12 +245,20 @@ func (c *ClusterUpConfig) Bind(flags *pflag.FlagSet) {
 	flags.StringArrayVarP(&c.Environment, "env", "e", c.Environment, "Specify a key-value pair for an environment variable to set on OpenShift container")
 	flags.BoolVar(&c.ShouldInstallMetrics, "metrics", false, "Install metrics (experimental)")
 	flags.BoolVar(&c.ShouldInstallLogging, "logging", false, "Install logging (experimental)")
-	flags.StringArrayVar(&c.AddComponents, "enable", c.AddComponents, "Install additional components.")
+	flags.StringSliceVar(&c.UserEnabledComponents, "enable", c.UserEnabledComponents, fmt.Sprintf(""+
```

Comment:
I would call 'default-enabled' -> 'core' components.

Reply:
> I would call 'default-enabled' -> 'core' components.

I don't want to make that assertion yet.  The word choice of "core" is going to be very political.